src/chains/ethereum/ethereum/index.ts

```diff
@@ -5,5 +5,159 @@
  * @license MIT
  */
 
+import chalk from "chalk";
+import { TruffleColors } from "@ganache/colors";
+import { WEI } from "@ganache/utils";
+import { toChecksumAddress } from "ethereumjs-util";
+import { EthereumProvider } from "./src/provider";
+import { Connector } from "./src/connector";
+import { EthereumDefaults } from "@ganache/ethereum-options";
+import { Flavor, ServerOptionsConfig } from "@ganache/flavor";
+
 export * from "./src/connector";
 export * from "./src/api-types";
+
+function capitalizeFirstLetter(string: string) {
+  return string[0].toUpperCase() + string.slice(1);
+}
+function color(str: string) {
+  return chalk`{hex("${TruffleColors.porsche}") ${str}}`;
+}
+
+interface EthereumFlavor
+  extends Flavor<EthereumProvider, ServerOptionsConfig, EthereumDefaults> {
+  flavor: "ethereum";
+  Connector: typeof Connector;
+  serverOptions: ServerOptionsConfig;
+  initialize: typeof initialize;
+}
+
+const EthereumFlavor: EthereumFlavor = {
+  flavor: "ethereum",
```

Comment:
non-technical thought

I think flavor makes sense. For the uninitiated, they would understand flavors means types of something. Would it be worthwhile to alias the branding to the specific thing for clarity? E.g. support both `pluginName` & `flavor` and they do the same thing?

Does this translate well to non-English users?

Reply:
`npmPackageNameOrInstallationPath` is what `flavor` means :-p